index.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable brace-style, camelcase, semi */
+/* global R5 */
+
+module.exports = Mailer;
+
+if (!global.R5) {
+  global.R5 = {
+    out: console
+  }
+}
+
+let mailer = require('mailgun-js');
+let config = {
+  domain:  'funnode.com',
+  key: '131133',
+  live: process.env.NODE_ENV === 'production'
+};
+
+// Constructors
+
+function Mailer () {
```

Comment:
this needs to be updated to accept the config as a param instead of it being predefined